Sources/Foundation/Logger/GTMLogger.m

```diff
@@ -20,7 +20,11 @@
 #error "This file needs to be compiled with ARC enabled."
 #endif
 
+#if __has_include("Foundation/GTMLogger.h")
```

Comment:
Realizing this is still in _Draft_ form, which builds needed this?

Reply:
This is a leftover from an old change (applied in our fork), which we might not need if PR #539 goes well. I’ll revisit this when I mark this PR as ready for review.